Server/BackEndMonitoringServer/include/CHardwareInfoLifeCycle.h

```diff
@@ -0,0 +1,25 @@
+#pragma once
+
+#include "�HardwareStatusSpecification.h"
```

Comment:
Mistype, fix Cyrillic 'C'

Reply:
Are you sure that it's good idea? It could make some confusion

Reply:
Fixed, includes changed to forward declarations, [commit](https://github.com/ITA-Dnipro/BackEndMonitoring/commit/28775d4e4e211a6985cecfb05174a510800a77d4#diff-0544ae2661f9747e70ea10c500758935c3f7d93fc11a7c69c55cadb18011b360)